packages/exchange-connector/test/exchangeTools.test.ts

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright (c) 2019, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { RestApi, groupByCategory } from "../src";
+import _ from "lodash";
+
+import { expect, default as chai } from "chai";
+import chaiAsPromised from "chai-as-promised";
+
+before(() => {
+  chai.use(chaiAsPromised);
+});
+
+describe("Test groupByCategory method", () => {
+  const apis: RestApi[] = [
+    {
+      id: "8888888/test-api/1.0.0",
+      name: "Test API",
+      groupId: "8888888",
+      assetId: "test-api",
+      version: "1.0.0"
+    },
+    {
+      id: "8888888/test-api-2/1.0.0",
+      name: "Test API 2",
+      groupId: "8888888",
+      assetId: "test-api-2",
+      version: "1.0.0"
+    },
+    {
+      id: "8888888/test-api-3/1.0.0",
+      name: "Test API 3",
+      groupId: "8888888",
+      assetId: "test-api-3",
+      version: "1.0.0"
+    },
+    {
+      id: "8888888/test-api-4/1.0.0",
+      name: "Test API 4",
+      groupId: "8888888",
+      assetId: "test-api-4",
+      version: "1.0.0"
+    }
+  ];
+
+  it("Group Apis (All same category)", () => {
```

Comment:
Not clear on what the test is doing...Input is clear, expected result is missing..

Rename tests to 

"<<Expected Result>> on <<given input>>"

Reply:
Rename tests to?  Can you provide a most specific example please?

Reply:
weird, I had the following in the comment,

Rename tests to 

"Expected Result on given input".